### ISSUE TITLE
Make it more obvious which comment box belongs to which question

### ISSUE THREAD

**Issue by fonglh** [Coursemology/coursemology2#3194]
![image](https://user-images.githubusercontent.com/1902527/48193734-232bdc80-e386-11e8-9b40-693a218192af.png)

**Comment by trangmei** [comment]
@fonglh what class did you add to get the after?

**Comment by fonglh** [comment]
The screenshot was from a mockup done by @aaronhojw.

**Comment by trangmei** [comment]
After changes are applied
<img width="467" alt="screen shot 2019-02-08 at 4 30 55 pm" src="https://user-images.githubusercontent.com/35135264/52467075-320d6f80-2bbf-11e9-9411-a008ecfc1a57.png">

**Comment by aaronhojw** [comment]
@trangmei Is the screenshot you posted on 8 Feb what it looks like finally?
Also, can help me understand why Statistics is circled too?

**Comment by trangmei** [comment]
@aaronhojw yes it's the final version, changes are circled. It's not yet deployed to production though.

'Statistics' has been changed from a card header to table header, so that it has the same format as 'Grade Summary' below.

You can still find the mismatched font size on production:
![image](https://user-images.githubusercontent.com/35135264/52608209-1d2d2680-2eb4-11e9-858a-2637ba5fc63d.png)

**Comment by aaronhojw** [comment]
@trangmei thanks for explaining! The change to Statistics looks good. 

As for the change to the style of Comments, I think it has resolved the main issue - thanks! 

I made another observation: Ideally, "Comments" should be nearer to the comment box than anything above. At the very least, the spacing between "Comments" and "grading" as well as "Comments" and "Enter your comment here" should be the same spacing. Perhaps this could be resolved by moving "Comments" down and the comment box elements up. 

I tried to simulate it in this mockup:
![red boxes](https://user-images.githubusercontent.com/45586446/52609143-3be0ec80-2eb7-11e9-8b1c-073b5d75f4ed.jpg)

Not sure if this will be different if it's not a teacher's view and if has proper question content.
